addons/sourcemod/scripting/jwp_mute.sp

```diff
@@ -50,6 +50,8 @@ public Action Event_OnRoundEnd(Event event, const char[] name, bool dontBroadcas
 			}
 		}
 	}
+
+	return Plugin_Continue;
```

Comment:
Do we really need it there?
There might be optimization, callback might return nothing, so we can replace **Action** to **void**

Reply:
Oh.. Night fixes are not the best thing. Fixed.